packages/flutter/test/material/page_test.dart

```diff
@@ -532,7 +532,9 @@ void main() {
     // Bring it exactly back such that there's nothing to animate when releasing.
     await gesture.moveBy(const Offset(-300, 0));
     await gesture.up();
+    // There will be a tiny animation upon release.
     await tester.pump();
+    await tester.pump(const Duration(milliseconds: 50));
```

Comment:
So this causes the side effect that there will always be a ballistic scroll at the end of a drag no matter where we dropped it off?

Reply:
yes, I try to update the test to make sure it just requires a small pump to finish the ballistic scroll that can be neglected by human eyes